telegram/ext/_jobqueue.py

```diff
@@ -106,7 +106,7 @@ def __init__(self) -> None:
 
         self._application: Optional[weakref.ReferenceType[Application]] = None
         self._executor = AsyncIOExecutor()
-        self.scheduler: AsyncIOScheduler = AsyncIOScheduler(**self.scheduler_configuration)
+        self.scheduler: "AsyncIOScheduler" = AsyncIOScheduler(**self.scheduler_configuration)
```

Comment:
ruff complains about this for me locally (i'm on py 3.13 so that's possibly why now)